### ISSUE TITLE
Implementing a carousel with info of our projects

### ISSUE THREAD

**Issue by govindgoel** [publiclab/community-toolbox#448]
### Please describe the problem (or idea)
Implementing a carousel with info of the projects where the dependency chart is added.
<img width="1440" alt="Screenshot 2021-03-27 at 10 58 42 PM" src="https://user-images.githubusercontent.com/52847415/112729047-35574700-8f50-11eb-900d-a2b3b33d823a.png">

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by govindgoel** [comment]
@jywarren Can you please give your opinion.

**Comment by karthikshetty03** [comment]
@govindgoel  Can I work on this issue?

**Comment by govindgoel** [comment]
@karthikshetty03 It has to be reviewed first.

**Comment by waridrox** [comment]
How about making the whole diagram interactive and implementing hoverable circles like this...

https://user-images.githubusercontent.com/58583793/112731228-fdee9780-8f5b-11eb-9dae-717412a7f380.mp4



and then we can also link the circles to respective github repos wherever applicable...

**Comment by govindgoel** [comment]
@waridrox we can also do that, what I thought is to provide short info regarding the projects.

**Comment by jywarren** [comment]
Hi all! I wonder - is it possible to pull project descriptions from the GitHub API? I think this is an interesting idea and could take some more discussion and research. I'm a bit more inclined to think about adding project descriptions into the list of projects by language than to take on a more complex interactive visualization within the scope of this project.  We don't want an overwhelming amount of information as it can be intimidating to newcomers! 

We are especially interested in different ways to display and sort collections of issues and contributors, like https://github.com/publiclab/community-toolbox/issues/315 -- so if you're interested in building things out a bit, we'd love your help and assistance on those kinds of issues! Thank you!!

**Comment by waridrox** [comment]
Yes @jywarren , I think it's possible to pull descriptions, here's a glimpse of the same 😄

<img width="1280" alt="desc" src="https://user-images.githubusercontent.com/58583793/113022512-50b99080-91a2-11eb-85bd-76eb8f2bc057.png">

**Comment by jywarren** [comment]
Oh, that's great. Shall we perhaps change the title to something like
"Expanding per-project info with descriptions"?

**Comment by govindgoel** [comment]
@jywarren Yeah I think that will be great to implement with this title
